pkg/services/rendering/rendering.go

```diff
@@ -83,19 +84,23 @@ func ProvideService(cfg *setting.Cfg, features featuremgmt.FeatureToggles, remot
 	switch {
 	case cfg.RendererUrl != "":
 		// RendererCallbackUrl has already been passed, it won't generate an error.
-		u, err := url.Parse(cfg.RendererCallbackUrl)
+		_, err := url.Parse(cfg.RendererCallbackUrl)
 		if err != nil {
+			logger.Warn("Image renderer callback url is not valid. " +
+				"Please provide a valid RendererCallbackUrl in the config. " +
+				"Read more at https://grafana.com/docs/grafana/latest/administration/image_rendering/")
 			return nil, err
 		}
 
 		sanitizeURL = getSanitizerURL(cfg.RendererUrl)
-		domain = u.Hostname()
+		domain = cfg.RendererCallbackUrl
```

Comment:
I don't think this should be changed as it's also sent to the image renderer to set the domain of the cookie. 

Also, this code change wouldn't have an impact in plugin mode because it's executed only when `cfg.RendererUrl` is set and it's set only when the image renderer is executed as a remote HTTP server.